### ISSUE TITLE
N/A(Missing driver - adodb-n/a.inc.php) Error while visting project details page

### ISSUE THREAD

**Issue by mayankSS** [reportico-web/yii2-reportico#2]
Missing file: ../projects/yii2-base/vendor/reportico/yii2-reportico/components/reportico_adodb/drivers/adodb-n/a.inc.php
ADONewConnection: Unable to load database driver ''

The issue is:

When any new project is created using project creation page, then config.php file gets generated  for each project.

This config.php has SW_DB_DRIVER parameter, it is always N/A for existing as well as for different connections(checked for pgsql and mysql).
define('SW_DB_DRIVER', 'N/A');

As per my understanding so far, it should be replaced with appropriate driver name like mysql.pgsql etc.

I am not sure why is it being generated as N/A. Just for info, I haven't added anything else other than yii2 base and yii2-reportico. There are no changes in yii2-reportico library as well.

Any help will be highly appreciated.

**Comment by reportico-web** [comment]
Hi there,

thank you for the post .. i have released a 4.3.1 version now with a simple fix for this.

This is a simple single line fix to 
yii2-reportico/components/projects/admin/configureproject.xml.php
setting the SW_DB_DRIVER after the TYPE definition

```
$configparams["SW_DB_DRIVER"] = $configparams["SW_DB_TYPE"];
```

Let me know or not if this helps.

Peter

**Comment by mayankSS** [comment]
Thanks, it is working now.
